doc/whatsnew/2/2.15/summary.rst

```diff
@@ -0,0 +1,26 @@
+:Release: 2.14
```

Comment:
```suggestion
:Release: 2.15
```

Reply:
Regarding the whatsnew for 2.15 I was thinking of discussing the new format here : https://github.com/PyCQA/pylint/pull/6688/files

Let me know what you think :)